### DISCUSSION TITLE
Inquiry on Obtaining Extreme Rays in YALMIP with Gurobi or CPLEX.

### DISCUSSION THREAD

**Discussion by dreamersliu** [yalmip/YALMIP#1423] [General discussion]
Dear Professor, 
      I am interested in extracting extreme rays in YALMIP with Gurobi or CPLEX. Any insights or references you could provide would be greatly appreciated. Thank you very much for your assistance.

![ray](https://github.com/user-attachments/assets/c577efab-9305-42bd-803a-1d05cae374c1)

      Best regards,
      Jing Liu

**Comment by johanlofberg** [comment]
https://yalmip.github.io/Extracting

**Reply by dreamersliu** [reply]
Thank you, Professor.
